auslib/web/admin/views/scheduled_changes.py

```diff
@@ -123,7 +121,7 @@ def _delete(self, sc_id, transaction, changed_by):
         if not sc:
             return Response(status=404, response="Scheduled change does not exist")
 
-        form = DbEditableForm(request.args)
+        form = DbEditableForm(connexion.request.args)
```

Comment:
Removing flask.request caused the sc/:namespace/<int:sc_id> API operations to break since request package was no longer defined. This form is being used in the  ScheduledChangeView group of APIs. They haven't been migrated in this commit.